spec/javascripts/item_selector_filtering_spec.js

```diff
@@ -25,7 +25,7 @@ describe('Item Selector Filtering', () => {
     });
 
     it('is a List', () => {
-      expect(itemSelectorFiltering.listPlugin()).toEqual(jasmine.any(List));
```

Comment:
I honestly don't know how this was passing before since we don't have jasmine as a dependency.